### ISSUE TITLE
Make a separate file to store library version

### ISSUE THREAD

**Issue by millionsun93** [Innovatube/android-boilerplate#4]
Currently, Version of build tool, support libraries, and 3rd libraries are out dated.If we want to update support version, we must edit multiple lines.
It's better if we create separate  file to store library version, build tool version ....(like inploi).

**Comment by toidv** [comment]
Definitely, we can do it. Separate all dependency libraries to a gradle file help to easily manage.
